models/cube.js

```diff
@@ -189,6 +189,6 @@ const Cube = mongoose.model('Cube', cubeSchema);
 
 Cube.LAYOUT_FIELDS = '_id owner name type card_count overrideCategory categoryOverride categoryPrefixes';
 Cube.PREVIEW_FIELDS =
-  '_id shortId urlAlias name card_count type overrideCategory categoryOverride categoryPrefixes image_name image_artist image_uri owner owner_name';
+  '_id shortId urlAlias name card_count type overrideCategory categoryOverride categoryPrefixes image_name image_artist image_uri owner owner_name image_uri';
```

Comment:
`PREVIEW_FIELDS` was modified, but `LAYOUT_FIELDS` is actually used for database connections in `cube_routes`.

Reply:
`PREVIEW_FIELDS` also already contains 'image_uri'